### ISSUE TITLE
Error: ETIMEDOUT

### ISSUE THREAD

**Issue by AdamMagaluk** [zettajs/zetta-hue-driver#5]
```js
/root/zetta/robotics-demo/node_modules/zetta-hue-driver/node_modules/node-hue-api/node_modules/q/q.js:126
                    throw e;
                          ^
Error: ETIMEDOUT
    at null._onTimeout (/root/zetta/robotics-demo/node_modules/zetta-hue-driver/node_modules/node-hue-api/node_modules/request-util/node_modules/request/request.js:679:15)
    at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)
```

**Comment by anandmahato29** [comment]
Did you solved it

**Comment by AdamMagaluk** [comment]
Yes i belive so. Fixed in af7f8fe3dd7b5640c086ae851080d89e3e4d9213
